subsys/bluetooth/shell/bt.c

```diff
@@ -731,6 +731,11 @@ static int cmd_init(const struct shell *sh, size_t argc, char *argv[])
 	return err;
 }
 
+static int cmd_disable(const struct shell *sh, size_t argc, char *argv[])
+{
+	return bt_disable();
```

Comment:
Context: I had deleted my comment here, but it seems @Thalley managed to see it before it was deleted.

I said that shell command handlers should not return errno values, but rather pick from the following list.

https://github.com/zephyrproject-rtos/zephyr/blob/30e4a4b71e3ecf54edef57f8c828a8f487068c1e/include/zephyr/shell/shell.h#L134-L137.

But then I was not so sure since two of choices are negated errno-values.